apps/dashboard/src/components/workflow-editor/nodes.tsx

```diff
@@ -235,7 +235,7 @@ export const AddNode = (_props: NodeProps<NodeType>) => {
   }
 
   return (
-    <div className="flex w-[300px] justify-center">
+    <div className="flex w-[300px] cursor-pointer justify-center">
```

Comment:
add step button cursor pointer fix: https://linear.app/novu/issue/NV-4684/workflow-overview-two-in-one-react-flow-and-cursor-on-hover